CMakeLists.txt

```diff
@@ -96,7 +96,11 @@ if(TODBC_DARWIN)
 endif()
 if(TODBC_WINDOWS)
   include_directories(C:/TDengine/include)
-  link_directories(C:/TDengine/driver)
+  if(TODBC_X86)
+    link_directories(C:/TDengine/taos_odbc/x86/lib)
```

Comment:
why install in such a folder but not %ProgramFiles% or %ProgramFiles(x86)%?

Reply:
This was initially the case, installed in the default path, but was rejected by the product manager, who recommended that all components be installed under the TDengine directory

Reply:
ok，I will try to separate the path selection between compile installation and package installation